content/en/logs/explorer/search.md

```diff
@@ -32,10 +32,64 @@ For example, to filter on logs produced by a web store service, with an error st
+
+{{< img src="logs/explorer/search/log_recent_searches.png" alt="The logs search bar showing `service:web-store status:error` as the query and recent searches for different web-store service errors as autocomplete options" style="width:80%;">}}
+
+### Autocomplete Saved Views
+
+You can create Saved Views in the Log Explorer to save queries and additional context for future and centralized access. The search bar autosuggests Saved Views that match your input in the search bar. Saved Views are displayed in the same order in which they are positioned in the Saved Views panel, with Saved Views that have been favorited displayed first. The Saved View name, saved query, and profile picture of the user who last updated it are displayed in the dropdown. If a Saved View query is too long to be displayed in the dropdown, the full query is displayed in a tooltip on hover. The email of the user who last updated a Saved View is also displayed in a tooltip on hover over their profile picture.
+
+{{< img src="logs/explorer/search/log_autocomplete_saved_views.png" alt="The logs search bar showing `service:web-store status:error` as the query and Saved Views for different web-store service errors as autocomplete options" style="width:80%;">}}
+
 ## Search syntax
 
+Syntax highlighting clearly differentiates input types, such as keys (e.g. an attribute such as `@merchant_name`), values (e.g. the name of a particular merchant), free text (e.g. keywords in a log messages such as `responded 500`), and control characters (e.g. parantheses and colons). Status attributes are also highlighted in colors representing the status, such as red for `error` and blue for `info`.
+
+{{< img src="logs/explorer/search/log_syntax_highlighting.png" alt="The logs search bar showing `service:auth-dotnet status:error 500 (check-token OR create-user)` as the query with differentiable syntax highlighting" style="width:80%;">}}
+
+Clear error states inform you which part of the query contains syntax errors and how to remediate them. For example,
+- If you input the query `service:` with no value, the message "Missing value in key:value pair" is displayed when you hover over the query.
+- If you input brackets for a range query, but do not fill in the high and low values, the message "Expected term but end of input found" is displayed.
+- If you input multiple values for a log field but miss the closing parenthesis character, such as  `service:(web-store OR auth-dotnet`, the message `Missing closing parenthesis character` is displayed:
+
+{{< img src="logs/explorer/search/log_error_states.png" alt="The logs search bar showing `service:(web-store OR auth-dotnet` as the query with the message `Missing closing parenthesis character`" style="width:80%;">}}
+
 To start searching for logs and customizing the time frame in the Log Explorer, read the [Search Syntax documentation][3] and the [Custom Time Frames documentation][4].
 
+## Raw and user-friendly search mode
```

Comment:
Pranay said to change this to:
```suggestion
## Disable styling for search bar
```

Reply:
I think we should mention that users can switch to raw text mode explicitly.

Reply:
Updated to what was decided on